### DISCUSSION TITLE
Error: Command "npm install" exited with 1

### DISCUSSION THREAD

**Discussion by carmen41511** [vercel/community#4230] [Help]
### Summary

I am trying to deploy a website but it failed with Error: Command "npm install" exited with 1.

 When I run this project locally, I had to install packages with `npm i --legacy-peer-deps`, `npm install --legacy-peer-deps` or `yarn install --legacy-peer-deps`; and when I build app for production, I had to `npm run build` or `yarn build`. 

I tried to override the Build Command and Install Command with these command, but they all failed with different error.

I'm also wondering if it is because of the root directory too, currently I chose `blogger` as the root.

This is my github repo for the project: https://github.com/carmen41511/hexsphere-website

I attached an image of my configuration.

### Example

https://vercel.com/carmen41511/hexsphere-website/Buaodgots7QNfaZLsQixELgF9v5r

### Steps to Reproduce

![image](https://github.com/vercel/community/assets/53323902/53a2ddcb-a249-46da-a685-a3816fde9962)

**Comment by shadoath** [comment]
Try adding an environmental value of
‘VERCEL_CLI_VERSION’ to ‘vercel@31.1.1’
Or the latest version that worked for you. You can find it in the build logs. 
![IMG_2818](https://github.com/vercel/community/assets/1946020/54970780-fe09-4dc1-ae2d-941a5feaeba7)
